templates/typescript-app/tsconfig.json

```
@@ -10,6 +10,7 @@
       "es2016"
     ],
     "module": "CommonJS",
+    "noEmit": true,
```

```suggestion
    "noEmit": false,
```

Lets keep emitting `.js` files in case the app ends up being published.